backend/src/main/java/ubb/courses/backend/security/AuthorityInterceptor.java

```diff
@@ -39,6 +40,17 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             }
         }
 
+        IsStudent isStudent = handlerMethod.getMethodAnnotation(IsStudent.class);
+        
+        if (isStudent != null) {
+            if (securityService.isStudent()) {
+                return true;
+            } else {
+                response.setStatus(HttpStatus.FORBIDDEN.value());
+                return false;
+            }
```

Comment:
Else condition is unnecessary. Please change it everywhere.

Reply:
Done ;)